pkgs/tools/filesystems/btrfs-heatmap/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, lib
+, fetchFromGitHub
+, python3
+}:
+
+stdenv.mkDerivation rec {
+  pname = "btrfs-heatmap";
+  # -1 because the debian branch has the man page
+  version = "8-1";
+
+  src = fetchFromGitHub {
+    owner = "knorrie";
+    repo = "btrfs-heatmap";
+    rev = "debian/8-1_bpo9+1";
```

Comment:
Hardcoding the debian rev is a bit unfortunate. Its easy to miss when updating `version` and breaks automatic updates. Please
- create an upstream issue asking if the manpages could be included in the regular releases and link to it in a comment
- use `${version}` in the `rev` so  that it will at least fail loudly when someone changes `version`.

Reply:
switched to regular v8 release
fetching the man page separately
and opened an issue as you suggested